proposals/lip-0048.md

```diff
@@ -25,15 +25,57 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
 ## Rationale
 
-### Fee Token ID
+Fees for blockchain transactions are necessary for a variety of reasons, including state growth, node computation, and block space utilization.
 
-Each chain can configure the token used to pay fees. On the Lisk mainchain, the token used for transaction fees is the LSK token.
+The current implementation lacks an automated method for calculating fees in all of these cases. Instead, we automate the calculation of block space fees using a *minimum fee* approach, similar to Bitcoin's mechanism, and allow developers discretion to use the [`payFee`](#payfee) method for fee assignment in other situations, e.g., when transaction implies state growth or more significant computational effort required by a node.
 
-### Minimum Fee per Transaction
+### Transaction Fees
 
-As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from the transaction size. Chains can configure their minimum fee per byte, with a possibility to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block when the minimum fee per byte is set to zero.
+Each transaction in the Lisk ecosystem has a *transaction fee* associated with it, as defined in [LIP 0068][lip-0068]. Transaction fees prevent spamming a blockchain with unlimited transactions at zero cost. In addition, a part of the transaction fee is provided as a reward to the validators who operate the network and maintain its security, as we explain in detail below. 
 
-Chains can choose that this minimum fee gets either burned (to avoid that validators can send transactions in the blocks they generate without cost) or transferred to the specified pool address `ADDRESS_FEE_POOL`. The latter choice could be beneficial for sidechains, as fees collected in this way can be used as an incentive for validators and other protocol participants. In the Lisk mainchain, the minimum fee is always burned.
+#### Fee Token ID
+
+Each chain can configure the token used to pay fees, as defined by `TOKEN_ID_FEE`. On the Lisk mainchain, the token used for transaction fees is the LSK token.
+
+#### Minimum Fee per Transaction
+
+As introduced in [LIP 0013][lip-0013], all transactions must have a transaction fee that is greater than or equal to a *minimum fee* that is always deducted from the part of the fee that goes to the block generator. The minimum fee, unless it is zero, is intended to discourage validators from sending zero-cost transactions in the blocks they generate.
```

Comment:
I think also here we are confusing things. The minFee is intended to charge for blockspace. What discourages validators from sending transactions is that they don't get the minFee back in case they send a transaction in the block they generate.

Reply:
```suggestion
As introduced in [LIP 0013][lip-0013], all transactions must have a transaction fee that is greater than or equal to a *minimum fee*. This fee corresponds to a cost for using the blockspace and therefore depends on the size of the transaction.
```

Reply:
Then the discussion about burning the minFee could be moved to after defining this fee.

Reply:
Yes, I agree that the minFee discussion is a bit unprecise; I've rewritten it according to your suggestion.